### ISSUE TITLE
broken chains / freedom

### ISSUE THREAD

**Issue by g76r** [FortAwesome/Font-Awesome#10288]
I'm seeking for an icon to symbolize freedom / breaking free. Such as broken ankle chains. It would be nice if such an icon would be part of the zillions of new icons that will be part of fa5 :-)
Apart from that thanks for your past and present work.

**Comment by tagliala** [comment]
does http://fontawesome.io/icon/chain-broken/ fit your needs?

**Comment by g76r** [comment]
@tagliala frankly, http://fontawesome.io/icon/chain-broken/ carries no meaning if not related to http://fontawesome.io/icon/link/

therefore it's okay to use fa-chain-broken on a UI for a button which show either fa-chain-broken or fa-link, but alone as a symbol of freedom it's meaningless, it's just a strange rounded rectangle emiting light

i would like a freedom/breaking free icon

either something with more context, such as: https://thenounproject.com/term/freedom/89995/
or another way to symbolize it such as an open lock: http://www.myiconfinder.com/icon/unlock-free-freedom-hack-lock-open-password-secure-security/12100 (which is not perfect for freedom)
or maybe the statue of liberty https://openclipart.org/detail/254013/statue-of-liberty-silhouette (which is not a universal symbol for freedom and can symbolize NYC rather...)

I don't know which is the better one, but imo not fa-chain-broken

**Comment by tagliala** [comment]
open lock:
http://fontawesome.io/icon/unlock/
http://fontawesome.io/icon/unlock-alt/

we need a better title for this one, as per our [contributing guidelines](https://github.com/FortAwesome/Font-Awesome/blob/master/CONTRIBUTING.md#requesting-new-icons)

> Request concrete objects: it's harder to make an icon to represent happiness, it's easier to make a smiley face. ☺

**Comment by g76r** [comment]
okay fa-unlock-alt can more or less do the job, far better than fa-chain-broken, but I still prefer https://thenounproject.com/term/freedom/89995/ or https://thenounproject.com/term/freedom/61825/ so if need ideas for the 1400 new fa5 icons... :-)

**Comment by g76r** [comment]
> we need a better title for this one, as per our contributing guidelines

a better title or just one more keyword in the search tools

**Comment by tagliala** [comment]
@g76r , nope, the title should meet contributing guidelines, so Icon request: icon-*name*, where name is something concrete :)

**Comment by g76r** [comment]
@tagliala yes like "fa-broken-handcuffs" or "fa-broken-ankle-chains" as a title and "freedom" and "breaking free" as keywords for the icon in algolia online search engine :-)

**Comment by tcalice** [comment]
Perhaps an anthropomorphic approach would be useful. A bird and cage seem to be a universal symbol of captivity/freedom.  

Humble thoughts on how to visually represent captivity, liberation, and freedom.  A caged bird is captive, an open cage with bird is liberating, and an uncaged bird is freedom.

**Comment by thisolivier** [comment]
Building on @tcalice with the bird idea, I can see a bird cage loosing clarity at small sizes, maybe a bird released from hands? It would also work as a more general peace icon.
The cage or cuffs would need to have a closed and open icon pair made.

My suggestion would be to add a dove of peace [see here](https://cdn4.iconfinder.com/data/icons/human-values-solid/100/_-62-512.png), with a protest fist a la [socialist posters](http://www.freediskspaceapp.com/img/freedom-icon-inv@2x.png). I think this would serve the purposes of representing freedom and breaking free, but also giving much more scope for general peace/protest/conflict communication using FA.

**Comment by tcalice** [comment]
I agree with concern about small resolution issue.  There is an abstract need for icons that could provide a visual status on liberated/captive states.  

I have another anthropomorphic freedom analogy.  How about a fish?

I could imagine three basic visual states for fish freedom could be: 1) Captive fish in fishbowl, 2) transitioning fish in net, and 3) freedom swimming in sea. 

Or in reverse order

3) freedom swimming in sea, 2) transitioning fish in net, 1) Captive fish in fishbowl

**Comment by g76r** [comment]
@thisolivier I like the idea of a bird for freedom, however the [dove-with-an-olive-branch](https://cdn4.iconfinder.com/data/icons/human-values-solid/100/_-62-512.png) remind me more of the peace than of the freedom (not sure everybody on earth feel like me, of course). And the [raised-fist icon](http://www.freediskspaceapp.com/img/freedom-icon-inv@2x.png) is nice but (still, as for me) remind me more of revolution than freedom, and let be frank many people will see it as communism or leftism without any positive freedom connotation (maybe this is only true in my country, but I don't think so).

@tcalice I like the idea of multi-state multi-icons, but I would like that one of the icons/states can mean "freedom" or "set free" by itself, without the need of the other states. This is exactly the reason why I don't think [fa-chain-broken](http://fontawesome.io/icon/chain-broken/) fits the need, because by itself it's only a strange flashing rectangle, not a symbol for freedom.

**Comment by thisolivier** [comment]
@g76r Yeah I struggled with trying to pull from existing imagery. It's important to note that no icon is unambiguous, and each use case will define the context for how the icon is read (re leftism), hence trying to something which fits in more broadly with FA, as opposed to meets your initial idea.

That being said it does look like we are failing to focus on the core request, an icon representing attaining freedom, as the expression goes, from the shackles of oppression. @tcalice's suggestion is well considered I think, but the transitioning stage is too ambiguous- a fish being placed into water seems so abstract as to be almost symbolic not iconic! Bird from cage is my favourite idea out of all of this, but I can imagine there being problems drafting it at small sizes.

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂

**Comment by sensibleworld** [comment]
Hey there!

We've made the decision to move our icon requests flow to **GitHub Discussions**, as opposed to issue tracking. No more +1s or thumbs-ups needed. There's an upvote system built in.

As part of this transition, we are closing existing icon requests with lower comment or reaction counts, but **we are not saying no** to your request. (We know some of these have been open since before I had gray hair. And I've had gray hair for a LONG time.)

**Could you do us a favor? If this icon request is still relevant for you, please re-open it in Discussions** (and feel free to include a link back to this issue):
→ https://github.com/FortAwesome/Font-Awesome/discussions/new?category=icon-requests

And listen. We know this is annoying. We would batch convert them to discussions if we could, but Github's removed that ability earlier this year. If you need to vent, here are some pre-written responses you can use when opening the new discussion:

* Cool cool cool. No worries at all. I love doing things twice.
* If an issue closes after seven years and nobody responds, did it ever really exist?
* I see you have chosen violence today. Re-opening per instructions.
* What's next? I have to file a TPS report, too?
* I am filled with the fury of a thousand suns. This request already HAS an auto-response saying that you're refining the workflow. What do you mean you're refining your refinement? That's a bunch of bull-phooey.
* No biggie. Thanks for letting us know what's up.
